src/Footer.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Next time, I will use this technique of splitting the components up this way. Good job. Makes your file structure easy to navigate.